lms/services/blackboard_api/_schemas.py

```diff
@@ -8,46 +8,35 @@
 from lms.validation._base import RequestsResponseSchema
 
 
-class _FileSchema(Schema):
-    """Shared base schema for Blackboard file dicts."""
-
-    class Meta:
-        unknown = EXCLUDE
-
-    id = fields.Str(required=True)
-    display_name = fields.Str(data_key="name", required=True)
-    updated_at = fields.Str(data_key="modified", required=True)
-    mime_type = fields.Str(data_key="mimeType")
```

Comment:
The `data_key`'s here were removed which means that the schema no longer renames the fields from Blackboard